### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by mmhobi7**
Missed a few and I didn't rename abc or abcd, also need to fix links

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -173,15 +173,15 @@ class CMainParams : public CChainParams {
...
+        // core.cash maintained by Corely
         vSeeds.push_back(
-            CDNSSeedData("clashic.cash", "seeder.clashic.cash", true));
+            CDNSSeedData("core.cash", "seeder.core.cash", true));
         // bitcoincore.zone
         vSeeds.push_back(
             CDNSSeedData("bitcoincore.zone", "seeder.bitcoincore.zone", true));
         // truevisionofsatoshi.com
         vSeeds.push_back(
-            CDNSSeedData("clashic.org", "seeder-mainnet.clashic.org", true));
```
Comment (mmhobi7) [inline src/chainparams.cpp]: also?

File: src/chainparams.cpp [hunk 2]
```diff
@@ -173,15 +173,15 @@ class CMainParams : public CChainParams {
 
         // Note that of those with the service bits flag, most only support a
         // subset of possible options.
-        // clashic.cash maintained by Clashicly
+        // core.cash maintained by Corely
         vSeeds.push_back(
-            CDNSSeedData("clashic.cash", "seeder.clashic.cash", true));
+            CDNSSeedData("core.cash", "seeder.core.cash", true));
```
Comment (mmhobi7) [inline src/chainparams.cpp]: ?

File: src/chainparams.cpp [hunk 3]
```diff
@@ -352,9 +352,9 @@ class CTestNetParams : public CChainParams {
         // truevisionofsatoshi.com
         vSeeds.push_back(CDNSSeedData("truevisionofsatoshi.com",
                                       "seeder-testnet.truevisionofsatoshi.com", true));
-        // clashic.cash maintained by Clashicly
+        // core.cash maintained by Corely
         vSeeds.push_back(
-            CDNSSeedData("clashic.cash", "testnet-seeder.clashic.cash", true));
+            CDNSSeedData("core.cash", "testnet-seeder.core.cash", true));
```
Comment (mmhobi7) [inline src/chainparams.cpp]: ?

File: src/seeder/main.cpp [hunk 4]
```diff
@@ -444,12 +444,12 @@ extern "C" void *ThreadStats(void *) {
 }
 
 static const std::string mainnet_seeds[] = {
-    "seeder.clashic.cash", "seeder.bitcoincore.zone", "seeder-mainnet.truevisionofsatoshi.com",
+    "seeder.core.cash", "seeder.bitcoincore.zone", "seeder-mainnet.truevisionofsatoshi.com",
```
Comment (mmhobi7) [inline src/seeder/main.cpp:447]: I already reverted this in the part t2

**Comment by mmhobi7** [comment]
CLASHIC->CORE

**Comment by mmhobi7** [comment]
Donzeo
Docs and Issue template need work.

File: src/seeder/main.cpp (hunk 4 above)
Comment (corelyone) [reply src/seeder/main.cpp:447]: seeder.clashic.cash is a real Bitcoin Core Seeder Node domain.

**Comment by mmhobi7** [comment]
I reverted the server stuff in the part 2

**Comment by corelyone** [comment]
@Aaahh Will speak to the team about implications of binary name changes again. Many new adopters have gotten very comfortable with 'clashic-cli'. We don't forget the roots of the project were from the Bitcoin Clashic chain. We wanted to run side by side with any other client in harmony. But will open up the dialog to team and community.

I do humbly thank you for your contribution. Jump on the Discord group and let's chat about your talents and how we can advance the project further with your help.

**Comment by mmhobi7** [comment]
Discord link on the .org is broken, I’m on telegram @TheBitcoinCore
If there is going to be a move away from Clashic-cli it should be sooner rather than later, symlinks could be added while the transition occurs if done.

**Comment by corelyone** [comment]
Leaving binaries named clashic as-is for now.
Updated some text files in 0.16.2 to reflect @Aaahh 's finding of copy that should have been updated in the prior release.

**Pull request closed without merging**